stable/jenkins/templates/test-configmap.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
 data:
   run.sh: |-
     @test "Testing Jenkins UI is accessible" {
-      curl --retry 12 --retry-delay 10 {{ .Release.Name }}-jenkins:8080{{ default "" .Values.Master.JenkinsUriPrefix }}/login
+      curl --retry 12 --retry-delay 10 {{ .Release.Name }}-jenkins:8080{{ default "" .Values.master.uriPrefix }}/login
```

Comment:
Add an empty default in `values.yaml`, then you don't need to specify a default here:

`{{ .Values.master.uriPrefix }}`